### ISSUE TITLE
Please put Episode's title and description with thumbnail

### ISSUE THREAD

**Issue by Bungeexgum** [deceptions/no#14]
This app would be perfect if they provide these features

**Comment by deceptions** [comment]
Gogoanime does not provide it, so I will have to use another service to get those. I might look into if I manage to make the Anilist connection to Gogoanime reliable.

**Comment by ty-coon** [comment]
> Gogoanime does not provide it, so I will have to use another service to get those. I might look into if I manage to make the Anilist connection to Gogoanime reliable.

Yes anilist api can be used to override gogoanime, they have kinda same pattern for episodes, just gotta have to mange multiple seasons
